c7n/resources/lex.py

```diff
@@ -19,3 +22,55 @@ class resource_type(query.TypeInfo):
         permissions_augment = ("lex:ListTagsForResource",)
 
     source_mapping = {"describe": query.DescribeWithResourceTags, "config": query.ConfigSource}
+
+
+@resources.register("lexv2-bot")
+class LexV2Bot(QueryResourceManager):
+    class resource_type(query.TypeInfo):
+        service = "lexv2-models"
+        enum_spec = ('list_bots', 'botSummaries', None)
+        arn_type = "bot"
+        arn_service = "lex"
+        id = "botId"
+        name = "botName"
+        cfn_type = config_type = "AWS::Lex::Bot"
+        universal_taggable = object()
+        permission_prefix = "lex"
+        permissions_augment = ("lex:ListTagsForResource",)
```

Comment:
nit: afaict we don't need this permission if we're using the bulk tagging api

```suggestion
```

Reply:
Ah, good catch! We're using the resource group tagging api here.